src/coins.h

```diff
@@ -419,6 +433,42 @@ class CCoinsView
     virtual ~CCoinsView() {}
 };
 
+class SubtreeCache {
```

Comment:
I think it would be useful to have a comment explaining how this class is intended to work.